Looped/FlowControllers/CreateFlowController.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+class CreateFlowController: FlowController {
+    
+    let navigationController: UINavigationController
+    let root: CreateViewController
+    
+    init(navigationController: UINavigationController, rootViewController: CreateViewController) {
```

Comment:
Zastanawia mnie po co initujesz to navigation controllerem.

To co chcesz w zasadzie osiagnac to fakt, ze kazdy flow controller bedzie odpowiadal za jeden ekran i cale flow w nim zawarte. 
Jesli chcesz przekazywac navigation controller to musisz go stworzyc w tabbarze i potem uzywac. Ja mysle, ze w tej sytuacji mozna zrobic tak:


```swift

    lazy var rootViewController: UINavigationController = {
        navigationController.viewControllers = [rootControllersFactory.buildSplashViewController(delegate: self)]

        return navigationController
    }()

    private lazy var navigationController: UINavigationController = {
        return UINavigationController()
    }()

```

W sensie bedziesz mial rootViewController a navigationController bedzie prywatny dla tego flow kontrolerach. 
We flow kontrolerach  chodzi o to, ze sam viewcontroller jest glupi i nie decyduje co bedzie sie dzialo po kliknieciu.

Mozna zmienic definicje protokolu na :

```swift
protocol FlowController {
     var rootViewController: UIViewController { get }
 }
```